stripe/src/main/java/com/stripe/android/model/Address.kt

```diff
@@ -13,7 +13,7 @@ import java.util.Locale
 @Parcelize
 data class Address internal constructor(
     val city: String? = null,
-    val country: String? = null,
+    val country: String? = null, // two-character country code
```

Comment:
should we just rename this to `countryCode` ?

Reply:
We should, but it would be a breaking change